regolith/helpers/a_presentationhelper.py

```diff
@@ -112,15 +115,19 @@ def db_updater(self):
 
         if not rc.authors:
             try:
-                rc.authors = [rc.default_user_id]
+                authors = [rc.default_user_id]
             except AttributeError:
                 raise RuntimeError(
                     "Please set default user ID and user first name in config.json file"
```

Comment:
use this message:
"WARNING: no authors have been set.  please rerun specifying authors, or add your id, e.g., sbillinge, to the config.json file in ~/.config/.regolith"  

(please check the path is correct)

Reply:
done.